sympy/solvers/tests/test_solveset.py

```diff
@@ -756,6 +756,12 @@ def test_solve_trig():
     assert solveset(sin(y + a) - sin(y), a, domain=S.Reals) == \
         imageset(Lambda(n, 2*n*pi), S.Integers)
 
+def test_issue_13084():
```

Comment:
You can add more testcases for eq with log, exp , trig eq , etc. which was not solvable before. Try once.